src/pages/pyeongan/Login/Login.js

```diff
@@ -1,5 +1,147 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import './Login.scss';
+import { useNavigate } from 'react-router-dom';
 
 export default function LoginP() {
-  return <></>;
+  const navigate = useNavigate();
+
+  const [values, setValues] = useState({
+    userid: '',
+    userpwd: '',
+  });
+
+  const onChange = e => {
+    const { name, value } = e.target;
+    setValues({ ...values, [name]: value });
+  };
+
+  const { userid, userpwd } = values;
+
+  const clicklink = () => {
+    navigate('/mainP');
+  };
+
+  const reset = e =>
+    setValues({
+      userid: '',
+      userpwd: '',
+    });
+
+  const effectiveness = userid.indexOf('@') > 0 && userpwd.length >= 5;
```

Comment:
- 조건으로 활용할 것이기 때문에 number를 return하는 indexOf보다 boolean을 return하는 includes를 사용하는 게 직관적이고 명확
- boolean 값을 가지는 데이터는 앞에 is를 붙이는 boolean naming convention 따르기
```suggestion
  const isAllValid = userid.includes('@') && userpwd.length >= 5;
```